docs/architecture/concepts/ve_separation/runme.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+
+CONDA_ROOT='/home/merzky/.miniconda3/'
```

Comment:
should here be kept `CONDA_ROOT="$HOME/.miniconda3/"`?

Reply:
and in `test_conda.sh`? (kinda to have a "universal" test without changes on user side)